NEW MASTER - User Gravatar Added

ogarocious opened the pull request.

Commit:
User Gravatar Added

melchua commented:
Still breaking
![Screen Shot 2022-04-12 at 10 47 20 PM](https://user-images.githubusercontent.com/16279091/163108814-15947f4d-1aed-42c6-9462-d133f70af94f.png)

Commit:
Added some simple bootstrap elements

Added a container to event types index page and a table bootstrap class to the user index
